src/main/java/org/cactoos/text/TextOf.java

```diff
@@ -317,17 +318,11 @@ public TextOf(final String input) {
      * Ctor.
      *
      * @param iterable The iterable to convert to string
-     * @todo #1461:30min We want constructors with {@code Iterable<?>} and
-     *  {@code Iterator<Character>} to have same behaviour (simply concatenate list of strings).
-     *  To do that, we should change {@code Iterable<?>} to {@code Iterable<Character>}
-     *  and delegate the {@link Iterator} one to the {@link Iterable} one. After that, we need to
-     *  modify other parts of cactoos that relied on this in order to preserve their behaviour
-     *  by using {@link Joined} and {@link Concatenated}.
-     */
-    public TextOf(final Iterable<?> iterable) {
+     */
+    public TextOf(final Iterable<Character> iterable) {
         super(
             new Joined(
-                ", ",
+                "",
```

Comment:
Should I add a ticket for moving this functionality to `Concatenated` (create `Concatenated(Iterable<Text>)`) to avoid using `Joined("", ...)`? WDYT?

Reply:
@DmitryBarskov you can either add a todo (see https://github.com/yegor256/pdd for how to format a todo, also there are many examples in the codebase) here or directly improve `Concatenated` as part of this PR, depending on how time you want to spend on this